doc/nrf/releases/release-notes-latest.rst

```diff
@@ -171,9 +270,17 @@ The following list summarizes the most important changes inherited from upstream
     * Changed the nRF UART nrfx drivers (uart_nrfx_uart/uarte) to use the DT ``hw-flow-control`` property instead of Kconfig options.
     * Fixed disabling of the TX interrupt in the uart_nrfx_uart driver.
     * Fixed the uart_nrfx_uarte driver to prevent spurious :cpp:enumerator:`UART_RX_BUF_REQUEST <uart_interface::UART_RX_BUF_REQUEST>` events.
+    * Removed counters reset from :cpp:func:`uart_rx_enable` in the nrf_uarte driver.
```

Comment:
Is "uart_rx_enable" correct?

Reply:
yes, https://docs.zephyrproject.org/latest/reference/peripherals/uart.html?highlight=uart_rx_enable#c.uart_rx_enable